exports/@material-ui/core/4.10.1.json

```diff
@@ -113,5 +113,9 @@
   "./Tooltip": "./esm/Tooltip/Tooltip.js",
   "./Typography": "./esm/Typography/Typography.js",
   "./Unstable_TrapFocus": "./esm/Unstable_TrapFocus/Unstable_TrapFocus.js",
-  "./Zoom": "./esm/Zoom/Zoom.js"
+  "./Zoom": "./esm/Zoom/Zoom.js",
+  "./styles/*": "./styles/*.js",
+  "./utils/*": "./utils/*.js",
```

Comment:
same here, I think only this is supported:

```
"./utils": "./esm/utils/index.js"
```

Reply:
In a few of the `esm/` component files they import other styles and utils directly other than index. I think the entire folder is meant to be imported

Reply:
Unless in our export maps do we only need to export “public” exports? And while preparing the package anything goes?